.github/CONTRIBUTING.md

```diff
@@ -56,12 +56,12 @@ Prerequisites:
    annoyingly slow):
 
    ```shell-session
-   $ tox -e lint
-   lint create: .tox/lint
-   lint installdeps: pre-commit, pylint ~= 2.8.0, pylint-pytest < 1.1.0
-   lint installed: ...
-   lint run-test-pre: PYTHONHASHSEED='2351399476'
-   lint run-test: commands[0] | python -m pre_commit run ... --all-files -v
+   $ tox -e lint-vetting
+   lint-vetting create: .tox/lint-vetting
+   lint-vetting installdeps: pre-commit, pylint ~= 2.8.0, pylint-pytest < 1.1.0
+   lint-vetting installed: ...
+   lint-vetting run-test-pre: PYTHONHASHSEED='2351399476'
+   lint-vetting run-test: commands[0] | python -m pre_commit run ... --all-files -v
```

Comment:
This is why we needed to disable MD013 on code-blocks.

Reply:
Interesting. This is why I initially shortened the example. I suppose, this can be fixed in a follow-up.